ui/app/components/secret-edit.js

```diff
@@ -108,60 +88,59 @@ export default Component.extend(FocusOnInsertMixin, WithNavToNearestAncestor, {
     'model',
     'model.id',
     'mode'
-  ),
-  canDeleteSecretMetadata: alias('checkMetadataCapabilities.canDelete'),
-  canUpdateSecretMetadata: alias('checkMetadataCapabilities.canUpdate'),
-  canReadSecretMetadata: alias('checkMetadataCapabilities.canRead'),
+  )
+  checkMetadataCapabilities;
+  @alias('checkMetadataCapabilities.canDelete') canDeleteSecretMetadata;
+  @alias('checkMetadataCapabilities.canUpdate') canUpdateSecretMetadata;
+  @alias('checkMetadataCapabilities.canRead') canReadSecretMetadata;
 
-  requestInFlight: or('model.isLoading', 'model.isReloading', 'model.isSaving'),
+  @or('model.isLoading', 'model.isReloading', 'model.isSaving') requestInFlight;
+  @or('requestInFlight', 'model.isFolder', 'model.flagsIsInvalid') buttonDisabled;
 
-  buttonDisabled: or('requestInFlight', 'model.isFolder', 'model.flagsIsInvalid'),
-
-  modelForData: computed('isV2', 'model', function () {
```

Comment:
here will be the backport fix. I'll make this `computed('isV2', 'model', 'selectedVersion', function ...`